src/pages/UserPage/UserPage.tsx

```diff
@@ -130,13 +130,25 @@ const UserPage: FC = () => {
       name: 'notify_user',
       label: t('label.user_activation_notification'),
       ariaLabel: t('label.user_activation_notification'),
-      rules: {
-        required: true,
-      },
       className: classes.checkboxInputClass,
     },
   ]
 
+  const onArchiveSubmit: SubmitHandler<FormValues> = useCallback(async () => {
+    const payload: UserStatusType = { institution_user_id }
+    navigate('/settings/users')
+
+    try {
+      await archiveUser(payload)
+
+      showNotification({
+        type: NotificationTypes.Success,
+        title: t('notification.announcement'),
+        content: t('success.user_archived', { name }),
+      })
+    } catch (_) {}
```

Comment:
**[non-blocker]**

No error displayed, when user archiving fails with some validation error. You can leave this for now, I can add it in the "cleanup" branch